recipes/configure.rb

```diff
@@ -39,7 +39,7 @@
 end
 
 template start_script do
-  source 'start-nexus-iq-server.sh.erb'
```

Comment:
start-nexus-iq-server.sh.erb is ok

Reply:
Changed to use hyphens for file system names.